frontend/bonsai_app/blueprints/sample/templates/cards.html

```diff
@@ -1038,6 +1040,42 @@ <h6 class="fw-semibold">Comment</h6>
 {% endmacro %}
 
 
+{% macro emm_card(emm) %}
+{# Card for displaying EMM typing results #}
+<div class="card col-md-auto">
+    <div class="card-header">
+        EMMtyper
+        <a class="ps-2" href="https://github.com/MDU-PHL/emmtyper" target="_blank">
+            <i class="bi bi-box-arrow-up-right"></i>
+        </a>
+    </div>
+    <div class="card-body">
+        <table class="table table-sm">
+            <tbody>
+                <tr>
+                    <th scope="row">Type</th>
+                    <td>{{ emm.emmtype }}</td>
```

Comment:
`emmtype`, `emm_cluster` & `cluster_count` can be `None`. We're fine to display `None` here?